js/react_native/example/ios/Podfile

```diff
@@ -9,7 +9,10 @@ target 'OnnxruntimeModuleExample' do
   use_react_native!(:path => config["reactNativePath"])
 
   use_frameworks!
-  pod 'onnxruntime-mobile-c', :podspec => '../../onnxruntime-mobile-c.podspec'
+
+  if File.exist?('../../local_pods/onnxruntime-mobile-c.zip')
```

Comment:
Do we need this if statement? Since it will fail anyway if the zip does not exist

Reply:
When a zip pod doesn't exist, a remote pod will be used because onnxruntime-react-native.podspec has a dependency on onnxruntime-mobile-c, so a dev doesn't need to build onnxruntime locally.